### PR TITLE
clean frontend

### PR DESCRIPTION
**Pull request by jackharrhy**
https://user-images.githubusercontent.com/6486417/142355606-594d6d56-f702-4078-983e-e86fb32e32a4.mov

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+		<BrowserRouter>
+			<Provider value={client}>
+				<div className="w-11/12 sm:w-2/3 lg:w-7/12 xl:w-1/2 2xl:w-5/12 mx-auto my-3">
+					<div className="w-[3.5rem]">
+						<img src={grack} className="h-16 mb-1 mx-auto" />
+					</div>
+					<div className="flex gap-3">
+						<Link to="/">GrackDB</Link>
+						<Link to="/users">Users</Link>
+						<Link to="/projects">Projects</Link>
```
Comment (nint8835) [inline frontend/src/App.jsx]: Could a link to the GQL playground also be added here?

File: frontend/tailwind.config.js [hunk 2]
```diff
@@ -1,21 +1,7 @@
 module.exports = {
+	mode: 'jit',
```
Comment (nint8835) [inline frontend/tailwind.config.js]: Seems the indentation here is inconsistent - would it be possible to get a basic prettier config set up to auto-format everything?

**Review by nint8835 — CHANGES_REQUESTED** [review]

File: frontend/tailwind.config.js (hunk 2 above)
Comment (jackharrhy) [reply frontend/tailwind.config.js]: done

File: frontend/src/App.jsx (hunk 1 above)
Comment (jackharrhy) [reply frontend/src/App.jsx]: done

File: api/api.go [hunk 3]
```diff
@@ -128,7 +128,10 @@ func StartApi() error {
 	srv := handler.NewDefaultServer(graphql.NewSchema(entClient))
 	srv.Use(entgql.Transactioner{TxOpener: entClient})
 
-	app.GET("/", playgroundHandler())
+	app.Static("/assets", "./assets")
```
Comment (nint8835) [inline api/api.go:131]: Could this use an embedded fs?

File: api/api.go [hunk 4]
```diff
@@ -128,7 +128,10 @@ func StartApi() error {
...
+	app.StaticFile("/", "./index.html")
```
Comment (nint8835) [inline api/api.go:132]: I believe this isn't a catch all - if I refresh a page I just get a 404

File: api/api.go (hunk 3 above)
Comment (nint8835) [reply api/api.go:131]: Would it also be possible to make this work locally?

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  grackdb:
+    image: fogo-sh/grackdb
+    build: ./.
+    ports:
+      - "8082:8081"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```
Comment (nint8835) [inline docker-compose.yml]: Does this require the docker sock?

**Review by nint8835 — CHANGES_REQUESTED** [review]

File: api/api.go (hunk 4 above)
Comment (jackharrhy) [reply api/api.go:132]: will be tackled by nint

File: api/api.go (hunk 3 above)
Comment (jackharrhy) [reply api/api.go:131]: will be tackled by nint

**Review by nint8835 — APPROVED** [review]

**Pull request merged**